about.html

```diff
@@ -0,0 +1,194 @@
+          <div class="collapse navbar-collapse justify-content-end navbar-nav-social" id="navbarNav">
+            <ul class="navbar-nav nav-bar-items-social">
+              <li class="nav-item">
+                <a class="nav-link active nav-icon" aria-current="page" href="#"><i class="bi bi-facebook"></i></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" href="#"><i class="bi bi-twitter"></i></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">Spanish</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">MyPage</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">Logout</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      
+      <div class="navbar navbar-expand-lg bg-body-tertiary navbar-menu">
+        <div class="container-fluid">
+          <div class="d-flex nav-bar-about">
+            <a class="navbar-brand" href="./index.html"><img src="https://wttwebcmsprod.blob.core.windows.net/eventimages/Singapore_Smash_2022.png" class="wttlogo"></a>
+            <ul class="navbar-nav nav-menu">
+              <li class="nav-item nav-item-home">
+                <a class="nav-link link-menu about-link" aria-current="page" href="./index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#join">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="./index.html#speakers">Players</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu link-campaign" href="#">LIVE</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+
+  <main class="main">
+    <section class="intro-about">
+      <div class="intro">
+        <h2 class="greet-about">EXPERIENCE THE SINGAPORE SMASH IN STYLE</h2>
+        <h1 class="title-about">Singapore Smash 2023<br>World Table Tennis 2023</h1>
+        <div class="bar-about-main"></div>
+        <div class="about">
+          <p class="description-about">
+            Men's and Women's Singles will feature 64 players each in the Main Draw. 50 players will be automatically entered based on ITTF World Table Tennis Ranking. <br>
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, your nav padding, your background design, text style, and alignment of elements do not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 
  
  ### Your Design
  ![image](https://user-images.githubusercontent.com/92303891/227748590-112b877a-f22c-4a1c-a81b-d1caae24af1d.png)
  
  ### Cindy's design
  ![image](https://user-images.githubusercontent.com/92303891/227748640-7dd1a7fb-80b2-4c11-94b0-afdc81999e90.png)